wordpress-mobile/WordPress-Android
# Convert me fragment to kotlin

In order to remove view memory leaks and usage of an inner async task I've converted the `MeFragment` to kotlin. There are mostly no functionality changes. The only change is replacing the async task with a coroutine in a view model.

To test:
- Go to the Me screen and stress test it
- Try log out of WP.com
- The progress dialog is shown
- You're correctly logged out

PR submission checklist:
